pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go

```diff
@@ -34,6 +34,42 @@ func (s ResourceStatus) String() string {
 	return names[s]
 }
 
+func Namespace(n NFD) (ResourceStatus, error) {
+
+	state := n.idx
+	obj := n.resources[state].Namespace
+
+	if len(n.ins.Spec.OperandNamespace) != 0 {
+		obj.SetName(n.ins.Spec.OperandNamespace)
+	}
+
+	found := &corev1.Namespace{}
+
```

Comment:
Sometimes this extra `\n` is here, sometimes it's not. Lets keep it consistent.

Reply:
Got it fixed.

Reply:
Got it, thanks.